app/src/main/java/org/piwigo/io/repository/UserRepository.java

```diff
@@ -43,11 +46,19 @@
         String url = userManager.getSiteUrl(account);
         String username = userManager.getUsername(account);
         String password = userManager.getPassword(account);
-        Observable<LoginResponse> result = login(url, username, password);
+        Observable<LoginResponse> result = null;
+        try {
+            result = login(url, username, password);
+        } catch (ExecutionException e) {
+            e.printStackTrace();
```

Comment:
maybe you wanna look at https://stackoverflow.com/questions/2072148/should-i-remove-e-printstacktrace-from-my-code-before-publishing